tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -97,5 +98,9 @@ def remove_resource(self, name, **kwargs):
             except Exception:
                 pass
 
-RandomNameResourceGroupPreparer = partial(ResourceGroupPreparer, random_name_enabled=True)
-CachedResourceGroupPreparer = partial(ResourceGroupPreparer, use_cache=True, random_name_enabled=True)
+    def sleep(self, duration=15):
```

Comment:
Nit:  Sleep_if_live?

Reply:
Actually not needed, I moved this to a different place so removing alltogether